pkg/logger/logger.go

```diff
@@ -13,115 +15,103 @@ const (
 	RESET  = "\x1b[33;0m"
 )
 
-type LogMessage struct {
-	Level   string
-	Message string
-}
-
 // Logger struct
 type Logger struct {
 	Verbosee bool
 	Debugg   bool
```

Comment:
I know it's out of the current PR scope

But please consider adding a comment if it's "normal" to have Verbosee and Debugg

I'm sure it's possible to find a better suffix. Here it looks like a typo

Reply:
I renamed them to `VerboseEnabled` and `DebugEnabled` - that seemed like a better name and avoiding that typo-esque suffix